test/misc.jl

```diff
@@ -42,8 +42,7 @@ end
             #  B₁(a, b) = B(a, b)
             a = rand(eltya)
             b = rand(eltyb)
-            @test logmvbeta(1, a, b) ≈ lgamma(a) + lgamma(b) - lgamma(a + b)
-            #@test logmvbeta(1, a, b) ≈ logbeta(a, b)  #  Whenever SpecialFunctions.jl updates
```

Comment:
Should we require a minimum version of SpecialFunctions?

Reply:
Is that what this is?

https://github.com/JuliaStats/StatsFuns.jl/blob/2ac7e3f848eeccea0024144d21a08c1cfb73d7a9/Project.toml#L9-L11

Reply:
Yes but do you know which version is actually required? I'm asking because you wrote "#  Whenever SpecialFunctions.jl updates". You could just try checking at SpecialFunctions 0.8 and running the tests.

Reply:
This comment is old, I switched to the new version in my second to last commit. `logbeta` was introduced in SpecialFunctions 0.8. The unit tests that just passed reflect all of this.